src/app/app-routing.module.ts

```diff
@@ -3,14 +3,16 @@ import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
 import { IndexComponent } from './components/index/index.component';
 import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+import { MyItinirariesComponent } from './components/itinerary/my-itiniraries/my-itiniraries.component';
 import { NotFoundComponent } from './components/not-found/not-found.component';
 import { LoginComponent } from './components/operations/login/login.component';
 import { RegisterComponent } from './components/operations/register/register.component';
 
 const routes: Routes = [
   //{path:'',component: AppComponent,pathMatch: 'full'},
   {path: '' , component: IndexComponent},
-  {path:':id/itineraries', component: ItinerarylistComponent,pathMatch: 'full'},
+  {path:'itinerarios/:id', component: ItinerarylistComponent,pathMatch: 'full'},
+  {path:'mis_itinerarios/:id', component: MyItinirariesComponent,pathMatch: 'full'},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
  {path:"mis_itinerarios/:id", component: MyItinirariesComponent,pathMatch: 'full'},
```